openmetadata-spec/src/main/resources/json/schema/entity/data/searchIndex.json

```
@@ -256,8 +262,20 @@
       "type": "string",
       "minLength": 1,
       "maxLength": 32
+    },
+    "indexType": {
+      "description": "Whether the entity is index or index template.",
+      "type": "string",
+      "javaType": "org.openmetadata.schema.entity.type.indexType",
```

"javaType": "org.openmetadata.schema.entity.type.SearchIndexType",
```suggestion
      "javaType": "org.openmetadata.schema.entity.type.SearchIndexType",
```

---

done